clang/tools/clang-offload-wrapper/ClangOffloadWrapper.cpp

```diff
@@ -676,7 +686,7 @@ class BinaryWrapper {
   // specified.
   Expected<std::pair<Constant *, Constant *>>
   addSYCLOffloadEntriesToModule(StringRef EntriesFile) {
-    if (EntriesFile.empty()) {
+    if (EntriesFile.empty() && !symPropReader) {
```

Comment:
Also it seems that `-entries` and `-sym-prop-bc-files` command line options are mutually exclusive. Corresponding explicit check should be performed. The same is for `-properties`.

Reply:
Add error checks for conflicting options.